app/src/main/java/com/mk/tasky/agenda/domain/model/AgendaPhoto.kt

```diff
@@ -0,0 +1,6 @@
+package com.mk.tasky.agenda.domain.model
+
+sealed class AgendaPhoto(val location: String) {
+    data class Local(val localUri: String) : AgendaPhoto(localUri)
+    data class Remote(val remoteUri: String, val key: String) : AgendaPhoto(remoteUri)
+}
```

Comment:
Great way to solve this! 💪

Reply:
Thanks!
Applied the knowledge of the other PR you reviewed :D